fgd/point/vgui/vgui_movie_display.fgd

```diff
@@ -1,13 +1,76 @@
 @PointClass base(BaseEntityPoint) 
 	appliesto(since_ASW) 
-	studio("models/editor/axis_helper_thick.mdl") 
+	studioprop("models/editor/axis_helper_thick.mdl") 
 	color(200 200 0) 
-	studio("models/editor/vgui_arrows.mdl") 
+	studioprop("models/editor/vgui_arrows.mdl") 
 	sphere(width) 
 	sphere(height) = vgui_movie_display: "Movie Display"
 	[
 	displaytext(string) : "Display Text"
-	moviefilename(string) : "Movie Filename" : "media/"
+	MovieFilename(choices) : "Movie to display" : "media/aperture_logo.bik" : "The movie to display. You can also type in the filename of a custom movie here." =
```

Comment:
This and `logic_playmovie` should probably both inherit from a `@BaseClass`, so they inherite the list of movie values.

Reply:
I actually tried doing that back when I originally added this to the combined FGDs. The filename is relative to a different location for each entity, so that doesn't work.